pkg/repro/repro.go

```diff
@@ -123,12 +123,12 @@ func Run(crashLog []byte, cfg *mgrconfig.Config, features *host.Features, report
 						continue
 					default:
 					}
+					time.Sleep(10 * time.Second)
```

Comment:
Looks impactful... Why did you move this sleep()?

Reply:
Is it a bug in qemu? We should have been killed and wait'ed for the previous qemu instance.
I assume this error will happen during fuzzing as well. On the other hand we don't need this sleep penalty for other VM types.
I think this better handled in vm/qemu/qemu.go. There is already a similar work around for "Could not set up host forwarding rule". I think that loop could handle this error as well.

Reply:
Yes. It is a qemu bug. 
Ok. I will add it to the workaround loop you suggest.